lib/versioning/npm/index.spec.ts

```diff
@@ -84,6 +84,16 @@ describe('semver.getNewValue()', () => {
       ['^1.2.3', '4.0.0', '^4.0.0'],
       ['^1.2.3', '4.5.6', '^4.0.0'],
       ['^1.0.0', '4.5.6', '^4.0.0'],
+
+      ['^0.2.3', '0.2.4', '^0.2.3'],
+      ['^2.3.0', '2.4.0', '^2.3.0'],
+      ['^2.3.4', '2.4.5', '^2.3.4'],
+      ['^0.0.1', '0.0.2', '^0.0.2'],
```

Comment:
I thought npm's semver treated `^0.0.1` as `>=0.0.1 <0.1.0`

Reply:
Good point, missed it

Reply:
https://github.com/npm/node-semver#caret-ranges-123-025-004
According to docs, it's `>=0.0.1 <0.0.2-0`

Reply:
Thanks, I was wrong. Leave as-is